website/files/models/base.py

```diff
@@ -192,7 +204,10 @@ def wrapped(self):
     def get_guid(self, create=False):
         """Attempt to find a Guid that points to this object.
         One will be created if requested.
-        :rtype: Guid
+
+        :param TrashedFileNode or StoredFileNode file_node
+        :param Boolean create: True will generate GUID if there isn't one
```

Comment:
I'm not sure. This is how the def was setup when I came to it. I only added the usage. My best guess is we should leave it be, as I'm not sure what all calls this.